CameraOverlay/topics.py

```diff
@@ -7,24 +7,25 @@ class NeatEnum(Enum):
     def __str__(self):
         return self.value
 
-
 class DAS(NeatEnum):
     """DAS MQTT Topics"""
     data = 'data'
     start = 'start'
     stop = 'stop'
 
-
 class PowerModel(NeatEnum):
     """Power Model MQTT Topics"""
     plan_name = 'power_model/plan_name'
     max_speed = 'power_model/max_speed'
     recommended_sp = 'power_model/recommended_SP'
     predicted_max_speed = 'power_model/predicted_max_speed'
 
-
 class Camera(NeatEnum):
     """Camera system MQTT Topics"""
     get_overlays = 'camera/get_overlays'
     set_overlays = 'camera/set_overlay'
     push_overlays = 'camera/push_overlays'
+
+class DAShboard(NeatEnum):
+        """DAShboard MQTT Topics"""
+        receive_message = 'v3/camera-system/primary/message'
```

Comment:
I think we're prefixing our MQTT topics with a `/`. Also, for consistency I think `camera` would be better than `camera-system`. Should also add some documentation about this new topic to [notion](https://www.notion.so/V3-MQTT-Topics-66e6715d0e1941ffaa82020e5868fbae) about what payload this topic should contain.